wallets/exodus-extension/src/extension/registry.ts

```diff
@@ -0,0 +1,28 @@
+import { Wallet } from '@cosmos-kit/core';
+import { GoDesktopDownload } from 'react-icons/go';
+import { RiChromeFill } from 'react-icons/ri';
+
+export const exodusExtensionInfo: Wallet = {
+  name: 'exodus-extension',
+  logo: 'https://exodus.com/brand/img/logo.svg',
```

Comment:
I tried the `packages/example` and added the Exodus wallet.  the icon is a little slow to load.